Cargo.toml

```diff
@@ -12,20 +12,18 @@ readme = "README.md"
 
 
 [features]
-default = [ "secp-recovery" ]
+default = [ ]
 base64 = [ "base64-compat" ]
-fuzztarget = ["secp256k1/fuzztarget", "bitcoin_hashes/fuzztarget"]
+fuzztarget = ["bitcoin_hashes/fuzztarget"]
 unstable = []
 rand = ["secp256k1/rand-std"]
 use-serde = ["serde", "bitcoin_hashes/serde", "secp256k1/serde"]
-secp-endomorphism = ["secp256k1/endomorphism"]
 secp-lowmemory = ["secp256k1/lowmemory"]
-secp-recovery = ["secp256k1/recovery"]
```

Comment:
This makes it semver-incompatible, so the major version should be bumped.

Reply:
True, I didn't think of that!

However, the bump to `secp256k1` 0.20 is also semver-incompatible because we are re-exporting the whole crate as `bitcoin::secp256k1`.

Reply:
There are many semver-incompatible things in master right now :). We will do a major version bump soon.